Dockerfile

```diff
@@ -39,4 +39,11 @@ RUN chmod a+x entry.sh
 
 ADD lib lib
 
+RUN apt-get update && apt-get install -y unzip
```

Comment:
This line will be moved to [L19](https://github.com/resin-io/resinos-tests/pull/205/files#diff-3254677a7917c6c01f55212f86c57fbfL19)

Reply:
Not sure I understand this comment. Are you moving it there now?

Reply:
Yes, I will move it but after the test will run correctly. Right now, this is just wip.